ckcp/test.sh

```diff
@@ -63,10 +64,71 @@ else
       # KUBECONFIG="$KUBECONFIG_KCP" kubectl get pods,pipelineruns
       KUBECONFIG="$KUBECONFIG_KCP" kubectl get pipelineruns
       echo "Print kube resources in the physical cluster (Note: physical cluster will not know what pipelinesruns are)"
-      
+
       KCP_NS_NAME="$(get_namespace)"
       kubectl get pods -n "$KCP_NS_NAME"
+    elif [ $arg == "chains" ]; then
+      echo "Arg $arg passed. Running chains test..."
+
+      ns="test-tekton-chains"
+      echo "Reset namespace '$ns'"
+      kubectl get namespace "$ns" >/dev/null && kubectl delete namespace "$ns"
+      kubectl create namespace "$ns"
+      kubectl apply -k "$SCRIPT_DIR/manifests/test/tekton-chains" -n "$ns"
+
+      # Wait everything to be fully setup for pipelines
+      while [ "$(kubectl get applications -n openshift-gitops tekton-chains -o json | jq -r ".status.sync.status")" != "Synced" ] || \
+        [ "$(kubectl get serviceaccounts -n test-tekton-chains | grep -cE "^pipeline ")" != "1" ]; do
+        echo -n "."
+        sleep 2
+      done
+      echo "OK"
+
+      # Trigger the pipeline
+      image_src="quay.io/aptible/alpine:latest"
+      image_name="$(basename "$image_src")"
+      image_dst="image-registry.openshift-image-registry.svc:5000/$ns/$image_name"
+      tkn -n "$ns" pipeline start simple-copy \
```

Comment:
we don't have tkn listed in the pre-requisites. So should we add that there?
(or)
just use standard kubectl commands to run the pipeline?

Reply:
Added to `DEPENDENCIES.md`